api/migrations/0211_profile_limit_access_to_guest.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.13 on 2024-06-20 10:02
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0210_profile_accepted_montandon_license_terms'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='profile',
+            name='limit_access_to_guest',
+            field=models.BooleanField(default=True, verbose_name='is not guest user'),
```

Comment:
We need to set False for all existing users, and then have default as True for all new users.

Reply:
@Rup-Narayan-Rajbanshi Please do confirm if this is working in properly in nightly?

Reply:
@thenav56 Verified. Working fine.